<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1919</width>
    <height>1045</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFrame" name="shadow">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(45, 48, 55);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1901</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top-left-radius : 10px;
border-top-right-radius : 10px;
background-color: #E7B18E;</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>30</y>
          <width>971</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Casta Thin</family>
          <pointsize>68</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: E7B18E;</string>
        </property>
        <property name="text">
         <string>Query-Document Similarity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_close">
        <property name="geometry">
         <rect>
          <x>1800</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FF605C;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_minimize">
        <property name="geometry">
         <rect>
          <x>1830</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FFBD44;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_gtw">
        <property name="geometry">
         <rect>
          <x>1860</x>
          <y>20</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #00CA4E;
border-radius: 9%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_back">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Ihome.png</pixmap>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #E7B18E;</string>
       </property>
       <property name="text">
        <string>Corpus</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_preprocess">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>610</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Preprocess</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_add">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>220</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/lenovo/Pictures/addLogo.png</normaloff>C:/Users/lenovo/Pictures/addLogo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>-570</x>
         <y>990</y>
         <width>771</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(110, 117, 134);</string>
       </property>
       <property name="text">
        <string>Final project by : Naufal Mufid F</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>1590</x>
         <y>580</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(45, 48, 55);
color: #E7B18E;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>651</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget
{
    color: rgb(231, 177, 142);;
    border: 1px solid rgb(231, 177, 142);;
    
    gridline-color: rgb(231, 177, 142);;
}
QTableWidget::item
{
  color: rgb(231, 177, 142);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <column>
        <property name="text">
         <string>File(s)</string>
        </property>
        <property name="background">
         <color>
          <red>231</red>
          <green>177</green>
          <blue>142</blue>
         </color>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>260</y>
         <width>1071</width>
         <height>771</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">

QTabWidget::tab-bar {
   border: 1px rgb(231, 177, 142);
   border-radius : 0px;
}

QTabBar::tab {
  background: rgb(174, 138, 116);
  color: rgb(45, 48, 55);
  padding: 10px;
   border-radius : 0px;
 }

 QTabBar::tab:selected {
  background: rgb(231, 177, 142);
   border-radius : 0px;
  
 }

QTabWidget::pane { 
   border: 1px rgb(231, 177, 142);
   background-color: rgb(45, 48, 55);
   border-radius : 0px;
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Guide</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>1051</width>
           <height>571</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E7B18E;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Selamat datang di aplikasi Query-Document Similairty!&lt;/span&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Tujuan dari aplikasi ini adalah untuk menghitung kemiripan antar query dengan (beberapa) dokumen lengkap dengan step preprocessing dan juga kemiripan antar dokumen pada corpus.&lt;/span&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Untuk langkah penggunaan aplikasi dapat dilihat di bawah.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;1. Klik button plus (+) untuk menambahkan dokumen ke dalam corpus.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;2. Pilih dokumen pada corpus yang akan dilakukan dikalkulasi kemiripannya dengan cara check dokumen tersebut&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;3. Klik button &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Preprocess&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; untuk melakukan tahapan preprocessing dan juga kalkulasi antar dokumen pada corpus.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;- Untuk lihat tahapan preprocessing, klik tab &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Preprocessing&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;- Untuk lihat hasil kalkulasi, klik tab &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Measurement&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;4. Masukkan query di kolom yang telah disediakan.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;5. Klik button &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Calculate&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; untuk melakukan proses kalkulasi kemiripan antara query dengan dokumen.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;- Untuk lihat hasil kalkulasi, klik tab &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Measurement&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Berikan saran agar aplikasi ini dapat lebih baik lagi ke depannya. Terima kasih!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Preprocessing</string>
        </attribute>
        <widget class="QTabWidget" name="isiPreprocessing">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1051</width>
           <height>711</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 0px;</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Measurement</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>360</y>
           <width>351</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Gramatika-Medium</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E7B18E;</string>
         </property>
         <property name="text">
          <string>Documents Similarity</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>731</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget
{
    color: rgb(231, 177, 142);;
    border: 1px solid rgb(231, 177, 142);
    
    gridline-color: rgb(231, 177, 142);;
}
QTableWidget::item
{
  color: rgb(231, 177, 142);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <column>
          <property name="text">
           <string>Document</string>
          </property>
          <property name="background">
           <color>
            <red>231</red>
            <green>177</green>
            <blue>142</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Similarity</string>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>-130</x>
           <y>30</y>
           <width>351</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Gramatika-Medium</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #E7B18E;</string>
         </property>
         <property name="text">
          <string>Query</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>420</y>
           <width>1071</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView
{
    color: black;
    border: 1px solid rgb(231, 177, 142);
    
    gridline-color: rgb(231, 177, 142);;
}
QTableView::item
{
  color: rgb(231, 177, 142);
}

QTableView::section {
    background-color: black;
    border: none;
}</string>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
        </widget>
        <zorder>label_7</zorder>
        <zorder>label_6</zorder>
        <zorder>tableWidget_3</zorder>
        <zorder>tableView</zorder>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>220</y>
         <width>531</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(231, 177, 142);
color: rgb(45, 48, 55);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>150</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #E7B18E;</string>
       </property>
       <property name="text">
        <string>Query</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_calculate">
       <property name="geometry">
        <rect>
         <x>1350</x>
         <y>220</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E7B18E;
border: 1px solid #E7B18E; 
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>990</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <zorder>widget_2</zorder>
      <zorder>label_4</zorder>
      <zorder>btn_preprocess</zorder>
      <zorder>btn_add</zorder>
      <zorder>label_14</zorder>
      <zorder>tableWidget</zorder>
      <zorder>label_5</zorder>
      <zorder>plainTextEdit</zorder>
      <zorder>btn_calculate</zorder>
      <zorder>textBrowser</zorder>
      <zorder>tabWidget</zorder>
      <zorder>label_credits</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
